src/Inkluzitron/Handlers/CommandsHandler.cs

```diff
@@ -82,7 +82,7 @@ private async Task CommandExecutedAsync(Optional<CommandInfo> command, ICommandC
 
                 // Reply to command message and mention only replied user
```

Comment:
Pokud tam má být false, tak by to chtělo upravit komentář. Takto je to matoucí.

Reply:
Opraveno b3e171f